lib/modules/dosomething/dosomething_signup/includes/Signup.php

```diff
@@ -110,7 +110,7 @@ private function build($data) {
       'id' => isset($northstar_user) ? $northstar_user->id : $data->field_northstar_id_value,
       'first_name' => dosomething_helpers_extract_field_data($drupal_user->field_first_name),
       'last_initial' => $last_initial,
-      'photo' => dosomething_helpers_extract_field_data($drupal_user->photo),
+      'photo' => isset($drupal_user->picture) ? $drupal_user->picture : NULL,
```

Comment:
awesome, thanks for everyone's input! Talked more with @DFurnes in slack but for documentation, here's the plan: 

1. update the code in phoenix for approval
2. open a new issue in northstar to address changes (adding user info, including avatar to pass back) 
3. make a new pr with changes in northstar
4. once approved, merge northstar and deploy
5. merge phoenix changes and deploy